ci-operator/jobs/codeready-toolchain/toolchain-common/codeready-toolchain-toolchain-common-master-postsubmits.yaml

```diff
@@ -0,0 +1,78 @@
+postsubmits:
+  codeready-toolchain/toolchain-common:
+  - agent: kubernetes
+    branches:
+    - ^master$
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    name: branch-ci-codeready-toolchain-toolchain-common-master-build
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --promote
```

Comment:
you don't need this, since there is not `promotion` configuration the ci-op config

Reply:
done in 50cc9a7cd